ms/del_test.go

```diff
@@ -12,6 +12,15 @@ import (
 	"testing"
 )
 
+func TestDelEmptyList(t *testing.T) {
+	k, _ := kuzzle.NewKuzzle(&internal.MockedConnection{}, nil)
+	memoryStorage := MemoryStorage.NewMs(k)
```

Comment:
why not use k.Ms ?

Reply:
Because these tests predates your changes adding a `Ms` singleton to any Kuzzle instance.

I'll make a global pass on the module (again) and clean unit tests. I was going to mark this PR as WIP anyway, as coding that module in CGo made several function spelling problems apparent

Reply:
@benoitvidis > aaaaaaand... done. I also use `nil` instead of an empty option object when relevant, and this made some bugs with nil options apparent, so I also fixed them